app/Models/Score/Best/Model.php

```diff
@@ -186,26 +186,29 @@ public function userRank($options)
 
     public function macroUserBest()
     {
-        return function ($query, $limit, $includes = []) {
-            $baseResult = (clone $query)->with($includes)->limit($limit * 3)->get();
+        return function ($query, $limit, $includes = [], $offset = 0) {
```

Comment:
the parameter order seems weird with `limit` and `offset` separated randomly by `includes`.